ui/apps/platform/src/Containers/VulnMgmt/widgets/MostCommonVulnerabilities.js

```diff
@@ -84,16 +84,23 @@ const MostCommonVulnerabilities = ({ entityContext, search, limit }) => {
 
             const parsedMessage = checkForPermissionErrorMessage(error, defaultMessage);
 
-            content = <NoResultsMessage message={parsedMessage} className="p-3" icon="warn" />;
+            content = (
+                <div className="w-full">
+                    <Alert variant="warning" isInline title={parsedMessage} component="p" />
+                </div>
```

Comment:
I'm not sure whether or not this would be usable generally across all of these changes, but would the following be equivalent and remove one more instance of Tailwind class names?
```suggestion
                 <Alert className="pf-v5-u-w-100" variant="warning" isInline title={parsedMessage} component="p" />
```

Reply:
Aha, I debated pro and con, but decided not to set more precedents for `className` in `Alert` element, which seems like should be rare in PatternFly-native pages.